src/api/db/data/20200627135426_request_delegate_attribute.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../attribute_descriptions'
```

Comment:
@adrianschroeter and @hennevogel, the date of this migration '20200627135426_request_delegate_attribute.rb' is on future for a reason?